Lib/_aix_support.py

```diff
@@ -0,0 +1,77 @@
+"""Shared AIX support functions."""
+
+import sys
+import subprocess
+from sysconfig import get_config_var
+
+"""
+AIX filesets are identified by four decimal values: V.R.M.F.
+V (version) and R (release) can be retreived using ``uname``
+M(aintenance) and F(ix) values are not available via uname
+For determining ABI compatibility the values V.R.M
+are essential - suplemented by a final value known as
+`builddate` that is expressed as YYWW (Year WeekofYear)
+
+The values for V.R.M of the system Python was built on are available in
+the variable sysconfig.get_config_var("BUILD_GNU_TYPE"). The system builddate
+value may be available in sys.config.get_config_var("AIX_BUILDDATE")
+When all of these values on a running system are greater or equal to the
+values of the build system then the bdist modules are binary compatible
+with the AIX kernel.
+
+For pep425 purposes the AIX platform tag becomes:
+"AIX-{}-{:04d}-{}".format(vmtl, builddate, bitsize)
+e.g., "AIX-6107-1415-32" for AIX 6.1 TL7 bd 1415, 32-bit
+and, "AIX-6107-1415-64" for AIX 6.1 TL7 bd 1415, 64-bit
+"""
+
+# _bd - set to impossible setting year98-week98 if None
+_tmp = str(get_config_var("AIX_BUILDDATE"))
+_bd = 9898 if (_tmp == "None") else int(_tmp)
+_bgt = get_config_var("BUILD_GNU_TYPE")
+_is_32bit = sys.maxsize == 2147483647
+
+
+def _aix_tag(vrtl, bd):
+    # type: (int, int) -> str
+    sz = 32 if _is_32bit else 64
+    return "AIX-{}-{:04d}-{}".format(vrtl, bd, sz)
+
+
+# compute vrtl from the VRMF string
+def _aix_vrtl(vrmf):
+    # type: (str) -> int
+    v, r, tl = vrmf.split(".")[:3]
+    return int("{}{}{:02d}".format(v[-1], r, int(tl)))
+
+
+def _aix_bosmp64():
+    # type: () -> List[str]
+    """
+    The fileset bos.mp64 is the AIX kernel. It's VRMF and builddate
+    reflect the current levels of the runtime environment.
+    """
+    tmp = subprocess.check_output(["/usr/bin/lslpp", "-Lqc", "bos.mp64"])
+    tmp = tmp.decode("utf-8").strip().split(":")  # type: ignore
+    # lpp, vrmf, bd = list(tmp[index] for index in [0, 2, -1])  # type: ignore
+    # e.g., ['bos.mp64', '7.1.4.34', '1806']
+    return list(tmp[index] for index in [0, 2, -1])
+
+
+def aix_platform():
+    # type: () -> str
+    lpp, vrmf, bd = _aix_bosmp64()
+    assert lpp == "bos.mp64", "%s != %s" % (lpp, "bos.mp64")
```

Comment:
Have you considered using f-strings for readability?
```suggestion
    assert lpp == "bos.mp64", f"{lpp} != bos.mp64"
```
Also, are you sure you want to do `assert` here? `assert`s are ripped off in optimized mode and may be skipped in runtime.

Reply:
The assert() is one of the things I added to get the code to pass "tox -e lint".
checking on ``lpp`` is just an additional assurance, but now I suppose both variable and assert can be removed.

As to f-strings - I have copied this over from what I have submitted to pypa/packaging - which also verifies against versions of Python that do not support/know f-strings. Trying, as much as possible, to have the code identical.

If a core developer insists that it must be an f-string here, then that is what it will be - and additional arguments (not supported on all platforms) for ``subprocess`` routines can be mandated for Python 3.9 (and I hope for Python 3.8).

Reply:
Also, removed the assert() as no longer necessary.